source/features/quick-repo-deletion.tsx

```diff
@@ -21,10 +21,14 @@ function handleToggle(event: delegate.Event<Event, HTMLDetailsElement>): void {
 		'.rgh-open-prs-of-forks' // PRs opened in the source repo
 	]);
 
-	if (hasContent && !confirm('This fork has open issues/PRs, are you sure you want to delete everything?')) {
+	if (hasContent && !confirm('This repo has open issues/PRs, are you sure you want to delete everything?')) {
 		// Close the <details> element again
 		event.delegateTarget.open = false;
 	} else {
+		if (!pageDetect.isForkedRepo() && !confirm('⚠️ ⚠️ ARE YOU TOTALLY SURE YOU WANT TO DELETE THIS REPOSITORY? ⚠️ ⚠️')) {
```

Comment:
This should be a plain else if, it can’t be nested or else `start` will still be called

Reply:
I changed the alert logic, tell me if that's what you intended for this feature.